apps/aevm/src/aevm_eeevm_state.erl

```diff
@@ -281,12 +281,17 @@ spend_gas(Gas, State) ->
         GasLeft when GasLeft < 0 -> aevm_eeevm:eval_error(out_of_gas)
     end.
 
+%% Of course WordSize should be 32 - but we have to be buggy until the
+%% first scheduled hard-fork.
 heap_to_heap(Type, Ptr, State) ->
+    heap_to_heap(Type, Ptr, State, 1).
```

Comment:
Maybe nicer to turn the default around, so that omitted word size means the correct behaviour and put the comment about the hard-fork at the buggy call instead.

Reply:
Yes, true... 

Did it this way since the buggy call is in another module and didn't want to change more than necessary... We still need to make changes here regardless.